app/assets/stylesheets/application.scss

```
@@ -3,17 +3,30 @@
 // Bootstrap body padding for fixed navbar
 body { padding-top: 80px; }
 
-.ngo-card {
-  margin-bottom: 20px;
-}
 
 .ngo-card {
   border: 1px solid rgba(0, 0, 0, 0.125);
   border-radius: 0.25rem;
   padding-left: 0px !important;
   height: 322px;
+  margin-bottom: 30px;
+  & .fantasy-name{
```

Poderia deixar um espaço entre o nome da classe e a chave?

```suggestion
  & .fantasy-name {
```